### ISSUE TITLE
Poets Pen socketed spells damage

### ISSUE THREAD

**Issue by CameronGoosen** [Openarl/PathOfBuilding#1050]
Is it possible to overwrite the cast speed of the socketed spell in Poets Pen to have the same cast speed as the attack speed of the attack skill in the chest peice so that we can more accurately see the damage of the socketed spells :D

**Comment by ephankim** [comment]
I also noticed that **attack** leech is not shown for spells socketed in Poets Pen's sockets while I see that life is actually leeched in the game.
It seems that Poets Pen transforms spells into attacks for that purpose

**Comment by Exportforce** [comment]
@ephankim no it doesn't!
You get leech for your attack which you are doing and that does damage.

**Comment by ephankim** [comment]
Yes, that's what I read in Path of Building calculations but I tried with lv1 Frenzy without support gems as trigger for a couple of AOE and got from 20% life to full life (about 4500) with a single attack against a mob

**Comment by ephankim** [comment]
With closer analysis, I guess the leech is coming from the Slayer's Endless Hunger ascendency skill "20% of Overkill Damage is Leeched as Life"
